### ISSUE TITLE
No incoming audio on incoming call in latest build.

### ISSUE THREAD

**Issue by driesken** [BelledonneCommunications/linphone-android#174]
Anyone else having an issue with the latest Linphone that there is no incoming audio on incoming calls? Audio in outgoing calls works fine in both directions. We're experiencing this on a Moto X Play (Android 6) and a Oneplus 3 (Android 8.0)

Version: Linphone Android 3.3.1, Linphone Core 3.12.0-273-g20efb4ad4

I'm seeing some strange errors in logcat:

> 12-06 15:54:16.099 19113 19113 I Linphone: This device (OnePlus/ONEPLUS A3003/msm8996) declares it has a built-in echo canceller.
12-06 15:54:16.099 19113 19113 I Linphone: No information available for [OnePlus/ONEPLUS A3003/msm8996],
12-06 15:54:16.099 19113 19113 I Linphone: Sound device information for [OnePlus/ONEPLUS A3003/msm8996] is: builtin=[yes], delay=[0] ms
12-06 15:54:16.107 19113 19113 I Linphone: This device (OnePlus/ONEPLUS A3003/msm8996) declares it has a built-in echo canceller.
12-06 15:54:16.107 19113 19113 I Linphone: No information available for [OnePlus/ONEPLUS A3003/msm8996],
12-06 15:54:16.107 19113 19113 I Linphone: Sound device information for [OnePlus/ONEPLUS A3003/msm8996] is: builtin=[yes], delay=[0] ms
12-06 15:54:16.107 19113 19113 I Linphone: MSVolume set gain to [0.000000 db], [1.000000] linear
12-06 15:54:16.110 19113 19113 I Linphone: This device (OnePlus/ONEPLUS A3003/msm8996) declares it has a built-in echo canceller.
12-06 15:54:16.110 19113 19113 I Linphone: No information available for [OnePlus/ONEPLUS A3003/msm8996],
12-06 15:54:16.110 19113 19113 I Linphone: Sound device information for [OnePlus/ONEPLUS A3003/msm8996] is: builtin=[yes], delay=[0] ms
12-06 15:54:16.110 19113 19113 I Linphone: MSVolume set gain to [0.000000 db], [1.000000] linear
12-06 15:54:16.110 19113 19113 I Linphone: configuring MSAlawDec:0x771f3fe880-->MSOpenSLESPlayer:0x77222ce880 from rate [8000] to rate [48000] and from channel [1] to channel [1]
12-06 15:54:16.110 19113 19113 I Linphone: ms_filter_link: MSAndSoundRead:0x771f24df80,0-->MSEqualizer:0x771f3fea80,0
12-06 15:54:16.110 19113 19113 I Linphone: ms_filter_link: MSEqualizer:0x771f3fea80,0-->MSVolume:0x771f21d300,0
12-06 15:54:16.110 19113 19113 I Linphone: ms_filter_link: MSVolume:0x771f21d300,0-->MSAudioMixer:0x771f3fea00,0
12-06 15:54:16.110 19113 19113 I Linphone: ms_filter_link: MSAudioMixer:0x771f3fea00,0-->MSAlawEnc:0x771f3fe800,0
12-06 15:54:16.110 19113 19113 I Linphone: ms_filter_link: MSAlawEnc:0x771f3fe800,0-->MSRtpSend:0x771f21d100,0
12-06 15:54:16.110 19113 19113 I Linphone: ms_filter_link: MSRtpRecv:0x771f267e00,0-->MSAlawDec:0x771f3fe880,0
12-06 15:54:16.110 19113 19113 I Linphone: ms_filter_link: MSAlawDec:0x771f3fe880,0-->MSGenericPLC:0x771f3feb80,0
12-06 15:54:16.110 19113 19113 I Linphone: ms_filter_link: MSGenericPLC:0x771f3feb80,0-->MSAudioFlowControl:0x771f3fec00,0
12-06 15:54:16.110 19113 19113 I Linphone: ms_filter_link: MSAudioFlowControl:0x771f3fec00,0-->MSDtmfGen:0x771f267e80,0
12-06 15:54:16.110 19113 19113 I Linphone: ms_filter_link: MSDtmfGen:0x771f267e80,0-->MSVolume:0x771f3fe900,0
12-06 15:54:16.110 19113 19113 I Linphone: ms_filter_link: MSVolume:0x771f3fe900,0-->MSEqualizer:0x771f3feb00,0
12-06 15:54:16.111 19113 19113 I Linphone: ms_filter_link: MSEqualizer:0x771f3feb00,0-->MSAudioMixer:0x771f3fec80,0
12-06 15:54:16.111 19113 19113 I Linphone: speex_lib_ctl init with neon ? 1
12-06 15:54:16.111 19113 19113 I Linphone: ms_filter_link: MSFilePlayer:0x771f3fed80,0-->MSResample:0x771c2f8a00,0
12-06 15:54:16.111 19113 19113 I Linphone: ms_filter_link: MSResample:0x771c2f8a00,0-->MSAudioMixer:0x771f3fec80,1
12-06 15:54:16.111 19113 19113 I Linphone: ms_filter_link: MSAudioMixer:0x771f3fec80,0-->MSResample:0x771f3fe980,0
12-06 15:54:16.111 19113 19113 I Linphone: ms_filter_link: MSResample:0x771f3fe980,0-->MSOpenSLESPlayer:0x77222ce880,0
12-06 15:54:16.111 19113 19113 I Linphone: Configuring recorder with [16] bits  rate [8000] nchanels [1] buff size [1280], chunk size [160]
12-06 15:54:16.111 19113 19113 I AudioRecord: set(): inputSource 7, sampleRate 8000, format 0x1, channelMask 0x10, frameCount 640, notificationFrames 0, sessionId 0, transferType 0, flags 0, opPackageName org.linphone uid -1, pid -1
12-06 15:54:16.112   893  6958 I APM_AudioPolicyManager: getInputForAttr() source 7, samplingRate 8000, format 1, channelMask 10,session 273, flags 0
12-06 15:54:16.112   534  1890 D audio_hw_primary: adev_open_input_stream: enter: sample_rate(8000) channel_mask(0x10) devices(0x80000004)        stream_handle(0xe7985800) io_handle(94) source(7) format 1
12-06 15:54:16.112   534  1890 D compress_voip: voice_extn_compress_voip_pcm_prop_check: VoIP PCM property is enabled
12-06 15:54:16.112   534  1890 D compress_voip: voice_extn_compress_voip_open_input_stream: enter
12-06 15:54:16.112   534  1890 D compress_voip: voip_set_mode: enter, format=1
12-06 15:54:16.112   534  1890 D compress_voip: voip_set_mode: Derived mode = 12
12-06 15:54:16.112   534  1890 W audio_hw_utils: audio_extn_utils_update_stream_input_app_type_cfg: App type could not be selected. Falling back to default
12-06 15:54:16.113   893 21061 I AudioFlinger: AudioFlinger's thread 0xed283bc0 tid=21061 ready to run
12-06 15:54:16.113   534   534 D audio_hw_primary: in_standby: enter: stream (0xe7985800) usecase(32: compress-voip-call)
12-06 15:54:16.114   534   534 D audio_hw_primary: in_standby: enter: stream (0xe7985800) usecase(32: compress-voip-call)
12-06 15:54:16.119   893   893 D AudioFlinger: Creating track with 2048 buffers bytes.format 1,mFrameSize 2,mFrameCount 640,roundup(frameCount) 1024
12-06 15:54:16.119   893   893 I AudioFlinger: package name:org.linphone
12-06 15:54:16.120   893   893 D AudioFlinger: ThreadBase::setParameters() AppRecTy=-1
12-06 15:54:16.120   534   534 D audio_hw_primary: in_set_parameters: enter: kvpairs=AppRecTy=-1
12-06 15:54:16.120   534   534 I audio_hw_primary: APP RECODER TYPE:-1
12-06 15:54:16.120   893   893 D AudioFlinger: opPackageName org.linphone
12-06 15:54:16.121 19113 19113 I Linphone: ms_ticker_set_time_func: ticker's time method updated.
12-06 15:54:16.121 19113 19113 I Linphone: AudioRecord.getAudioSessionId() returned 273
12-06 15:54:16.122 19113 21064 I Linphone: Start recording
12-06 15:54:16.123   893  6958 V AudioPolicyManagerCustom: startInput() input 94
12-06 15:54:16.124   534  1890 D audio_hw_primary: in_set_parameters: enter: kvpairs=input_source=7;routing=-2147483644
12-06 15:54:16.125   893  6958 V AudioPolicyManagerCustom: AudioPolicyManager::startInput() input source = 7
12-06 15:54:16.125   893  1523 I SoundTriggerHwService::Module: onCallbackEvent no clients
12-06 15:54:16.125   893  6958 D AudioPolicyManagerCustom: startInput() -- session 273 on input 94 input source = 7
12-06 15:54:16.127   534 21065 D compress_voip: voice_extn_compress_voip_start_input_stream: enter
12-06 15:54:16.127   534 21065 D compress_voip: voip_start_call: enter
12-06 15:54:16.127   534 21065 D audio_hw_primary: select_devices for use case (compress-voip-call)
12-06 15:54:16.127   534 21065 D msm8974_platform: platform_get_input_snd_device: enter: mode(3) str_bitwidth(16) out_device(0x1) source 7 in_device(0x4) channel_count (1) channel_mask (0x10)
12-06 15:54:16.127   534 21065 D audio_hw_primary: select_devices: out_snd_device(15: voice-handset) in_snd_device(92: voice-dmic-ef)
12-06 15:54:16.127   534 21065 I msm8974_platform: platform_check_and_set_codec_backend_cfg:becf: afe: bitwidth 16, samplerate 48000 channels 2, backend_idx 0 usecase = 32 device (voice-handset)
12-06 15:54:16.127   534 21065 D msm8974_platform: platform_split_snd_device: snd_device(15) num devices(1) new_snd_devices(0)
12-06 15:54:16.127   534 21065 I msm8974_platform: platform_check_and_set_codec_backend_cfg: new_snd_devices[0] is 15
12-06 15:54:16.127   534 21065 I msm8974_platform: platform_check_codec_backend_cfg:becf: afe: bitwidth 16, samplerate 48000 channels 2, backend_idx 0 usecase = 32 device (voice-handset)
12-06 15:54:16.127   534 21065 W msm8974_platform: platform_check_codec_backend_cfg:becf: afe:Use default bw and sr for voice/voip calls 
12-06 15:54:16.127   534 21065 D msm8974_platform: platform_check_codec_backend_cfg:becf: afe: playback on codec device not supporting native playback set default Sample Rate(48k)
12-06 15:54:16.127   534 21065 I msm8974_platform: platform_check_codec_backend_cfg:becf: afe: Codec selected backend: 0 updated bit width: 16 and sample rate: 48000
12-06 15:54:16.127   534 21065 D audio_hw_primary: check_usecases_codec_backend:becf: force routing 0
12-06 15:54:16.127   534 21065 D audio_hw_primary: check_usecases_codec_backend:becf: (47) check_usecases curr device: voice-handset, usecase device:handset backends match 1
12-06 15:54:16.127   534 21065 D audio_hw_primary: check_usecases_codec_backend:becf: check_usecases (low-latency-playback) is active on (handset) - disabling ..
12-06 15:54:16.127   534 21065 D audio_hw_primary: disable_audio_route: reset and update mixer path: low-latency-playback
12-06 15:54:16.129 19113 19113 I Linphone: AcousticEchoCanceler successfully created.
12-06 15:54:16.129 19113 19113 I Linphone: AcousticEchoCanceler enabled: 1
12-06 15:54:16.129 19113 19113 W Linphone: AcousticEchoCanceler already enabled
12-06 15:54:16.130 19113 19113 I Linphone: Initializing speex resampler in mode [voip] 
12-06 15:54:16.130 19113 19113 I Linphone: Initializing speex resampler in mode [voip] 
12-06 15:54:16.130 19113 19113 D         : PlayerBase::PlayerBase()
12-06 15:54:16.130 19113 19113 D         : TrackPlayerBase::TrackPlayerBase()
12-06 15:54:16.130 19113 19113 I libOpenSLES: Emulating old channel mask behavior (ignoring positional mask 0x4, using default mask 0x1 based on channel count of 1)
12-06 15:54:16.130 19113 19113 D AudioTrack: set(): streamType 2, sampleRate 48000, format 0x1, channelMask 0x1, frameCount 0, flags #104, notificationFrames 4294967294, sessionId 281, transferType 0, uid -1, pid -1, cbf 0
12-06 15:54:16.130 19113 19113 I AudioTrack: OPM,client expresses a preference for Fast!
12-06 15:54:16.130   893   893 V AudioPolicyManagerCustom: getOutput() The last flag  260
12-06 15:54:16.131   893   893 D AudioPolicyManagerCustom: getOutputForDevice() returns output 13 flag 260
12-06 15:54:16.131   893   893 D HalHidl : getParameters() keys is_direct_pcm_track, 
12-06 15:54:16.131   534  1240 D compress_voip: voice_extn_compress_voip_out_get_parameters: enter
12-06 15:54:16.131   893   893 W AudioFlinger: createTrack_l(): mismatch between requested flags (00000104) and output flags (00000006)
12-06 15:54:16.131   893   893 D AudioFlinger: AudioOut_D createTrack_l() stmType 2,typeVolume 0.000000,mute 0,fc 384,sId 281, flags 4
12-06 15:54:16.131   893   893 D AudioFlinger: Creating track with 1024 buffers bytes.format 1,mFrameSize 2,mFrameCount 384,roundup(frameCount) 512
12-06 15:54:16.131 19113 19113 I AudioTrack: AUDIO_OUTPUT_FLAG_FAST successful; frameCount 384 -> 384
12-06 15:54:16.131 19113 19113 D AudioTrack: Client defaulted notificationFrames to 192 for frameCount 384
12-06 15:54:16.133   532   532 W /system/bin/hw/android.hidl.allocator@1.0-service: ashmem_create_region(3840) returning hidl_memory(0x7c5a62c4e0, 3840)
12-06 15:54:16.135   532   532 W /system/bin/hw/android.hidl.allocator@1.0-service: ashmem_create_region(3840) returning hidl_memory(0x7c5a62c4e0, 3840)
12-06 15:54:16.136   893  1245 I APM_AudioPolicyManager: startOutput() output 13, stream 2, session 281
12-06 15:54:16.136   893  1245 I APM_AudioPolicyManager: setOutputDevice() device 0001 delayMs 0
12-06 15:54:16.136   893  1245 V AudioPolicyManagerCustom: handleIncallSonification() stream 2 starting 1 device 1 stateChange 0
12-06 15:54:16.136   893  1245 V AudioPolicyManagerCustom: handleIncallSonification() low visibility, muteCount 1
12-06 15:54:16.136   893  1245 I AudioPolicyManagerCustom: checkAndSetVolume:stream:2,index:0 device:1,delayMs:0,muteCount:0
12-06 15:54:16.136   893  1245 I AudioPolicyManagerCustom: checkAndSetVolume:stream:2,index:5 device:1,delayMs:0,muteCount:1
12-06 15:54:16.136   893  1245 D APM_AudioPolicyManager: startOutput()--
12-06 15:54:16.139 19113 19113 I Linphone: This device (OnePlus/ONEPLUS A3003/msm8996) declares it has a built-in echo canceller.
12-06 15:54:16.139 19113 19113 I Linphone: No information available for [OnePlus/ONEPLUS A3003/msm8996],
12-06 15:54:16.139 19113 19113 I Linphone: Sound device information for [OnePlus/ONEPLUS A3003/msm8996] is: builtin=[yes], delay=[0] ms
12-06 15:54:16.139 19113 19113 I Linphone: MSVolume set gain to [0.000000 db], [1.000000] linear
12-06 15:54:16.140 19113 19113 I Linphone: No valid video stream defined.
12-06 15:54:16.140 19113 19113 I Linphone: LinphoneCall[0x772236f000] : payload type 8 PCMA/8000 fmtp= added to frozen list.
12-06 15:54:16.140 19113 19113 I Linphone: LinphoneCall[0x772236f000] : payload type 0 PCMU/8000 fmtp= added to frozen list.
12-06 15:54:16.140 19113 19113 I Linphone: LinphoneCall[0x772236f000] : payload type 101 telephone-event/8000 fmtp= added to frozen list.
12-06 15:54:16.140 19113 19113 I Linphone: Call 0x772236f000: moving from state LinphoneCallConnected to LinphoneCallStreamsRunning
12-06 15:54:16.140 19113 19113 I Linphone: New call state [StreamsRunning]
12-06 15:54:16.141 19113 21057 I Linphone: Stun packet sent for session [0x77243e0400]
12-06 15:54:16.141 19113 19113 W Linphone: [AudioManager] already in MODE_IN_COMMUNICATION, skipping...
12-06 15:54:16.141 19113 21057 I Linphone: Stun packet sent on rtcp for session [0x77243e0400]
12-06 15:54:16.141 19113 21057 I Linphone: MSAudioMixer [0x771f3fec80] is entering bypass mode.
12-06 15:54:16.141 19113 19113 E Linphone: CallIncommingActivity - onCreate -  State.StreamsRunning - speaker = false
12-06 15:54:16.143 19113 19113 I AudioManager: setSpeakerphoneOn false java.lang.Throwable
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.media.AudioManager.setSpeakerphoneOn(AudioManager.java:1235)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at org.linphone.core.LinphoneCoreImpl.routeAudioToSpeakerHelper(LinphoneCoreImpl.java:446)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at org.linphone.core.LinphoneCoreImpl.enableSpeaker(LinphoneCoreImpl.java:457)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at org.linphone.CallIncomingActivity$5.callState(CallIncomingActivity.java:194)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at org.linphone.core.LinphoneCoreImpl.acceptCallWithParams(Native Method)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at org.linphone.core.LinphoneCoreImpl.acceptCallWithParams(LinphoneCoreImpl.java:975)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at org.linphone.LinphoneManager.acceptCallWithParams(LinphoneManager.java:1707)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at org.linphone.CallIncomingActivity.answer(CallIncomingActivity.java:304)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at org.linphone.CallIncomingActivity.access$700(CallIncomingActivity.java:48)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at org.linphone.CallIncomingActivity$2.onTouch(CallIncomingActivity.java:133)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.View.dispatchTouchEvent(View.java:11758)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2965)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2650)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2965)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2650)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2965)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2650)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2965)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2650)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2965)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2650)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:445)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1842)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.app.Activity.dispatchTouchEvent(Activity.java:3315)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:407)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.View.dispatchPointerEvent(View.java:12001)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4850)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4633)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4164)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4217)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4183)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4310)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4191)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4367)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4164)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4217)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4183)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4191)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4164)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:6738)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:6712)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:6658)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6841)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:186)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.InputEventReceiver.nativeConsumeBatchedInputEvents(Native Method)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.InputEventReceiver.consumeBatchedInputEvents(InputEventReceiver.java:177)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl.doConsumeBatchedInput(ViewRootImpl.java:6812)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$ConsumeBatchedInputRunnable.run(ViewRootImpl.java:6864)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:966)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.Choreographer.doCallbacks(Choreographer.java:778)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.Choreographer.doFrame(Choreographer.java:707)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:952)
12-06 15:54:16.143 19113 19113 I AudioManager: 	at android.os.H
12-06 15:54:16.144  1268 12140 D AudioService: isHasSpeakerAuthority APP keys: org.linphone,mIsTelName=false
12-06 15:54:16.144  1268 12140 D AudioService: isInCallState =false
12-06 15:54:16.145   893  6958 D AudioPolicyManagerCustom: setForceUse() usage 0, config 0, mPhoneState 3
12-06 15:54:16.145 19113 19113 I Linphone: Linphone core [0x772dc55a00] notified [call_state_changed]
12-06 15:54:16.145 19113 19113 I Linphone: Call answered
12-06 15:54:16.145 19113 19113 W Linphone: Routing audio to earpiece, disabling bluetooth audio route
12-06 15:54:16.147 19113 19113 I AudioManager: setSpeakerphoneOn false java.lang.Throwable
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.media.AudioManager.setSpeakerphoneOn(AudioManager.java:1235)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at org.linphone.core.LinphoneCoreImpl.routeAudioToSpeakerHelper(LinphoneCoreImpl.java:446)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at org.linphone.core.LinphoneCoreImpl.enableSpeaker(LinphoneCoreImpl.java:457)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at org.linphone.LinphoneManager.routeAudioToSpeakerHelper(LinphoneManager.java:257)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at org.linphone.LinphoneManager.routeAudioToReceiver(LinphoneManager.java:350)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at org.linphone.CallIncomingActivity.answer(CallIncomingActivity.java:311)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at org.linphone.CallIncomingActivity.access$700(CallIncomingActivity.java:48)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at org.linphone.CallIncomingActivity$2.onTouch(CallIncomingActivity.java:133)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.View.dispatchTouchEvent(View.java:11758)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2965)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2650)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2965)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2650)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2965)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2650)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2965)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2650)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2965)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2650)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:445)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1842)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.app.Activity.dispatchTouchEvent(Activity.java:3315)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:407)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.View.dispatchPointerEvent(View.java:12001)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4850)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4633)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4164)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4217)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4183)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4310)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4191)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4367)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4164)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4217)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4183)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4191)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4164)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:6738)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:6712)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:6658)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6841)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:186)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.InputEventReceiver.nativeConsumeBatchedInputEvents(Native Method)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.InputEventReceiver.consumeBatchedInputEvents(InputEventReceiver.java:177)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl.doConsumeBatchedInput(ViewRootImpl.java:6812)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.ViewRootImpl$ConsumeBatchedInputRunnable.run(ViewRootImpl.java:6864)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:966)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.Choreographer.doCallbacks(Choreographer.java:778)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.Choreographer.doFrame(Choreographer.java:707)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:952)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.os.Handler.handleCallback(Handler.java:789)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.os.Handler.dispatchMessage(Handler.java:98)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.os.Looper.loop(Looper.java:164)
12-06 15:54:16.147 19113 19113 I AudioManager: 	at android.app.Act
12-06 15:54:16.148  1268  2791 D AudioService: isHasSpeakerAuthority APP keys: org.linphone,mIsTelName=false
12-06 15:54:16.148  1268  2791 D AudioService: isInCallState =false
12-06 15:54:16.149   893  6958 D AudioPolicyManagerCustom: setForceUse() usage 0, config 0, mPhoneState 3
12-06 15:54:16.150  1268  2791 I ActivityManager: checkParallelAppControl, from:org.linphone, to:org.linphone, with act:null, callingUserId:0, toUserId:0
12-06 15:54:16.150  1268  2791 I ActivityManager: START u0 {cmp=org.linphone/.CallActivity} from uid 10339 pid 19113
12-06 15:54:16.151 19113 21057 I Linphone: MSAudioMixer [0x771f3fea00] is entering bypass mode.
12-06 15:54:16.152 19113 21057 I Linphone: MSAudioMixer [0x771f3fec80] is leaving bypass mode.
12-06 15:54:16.152  1268  2791 D ActivityTrigger: activityStartTrigger: Activity is Triggerred in full screen ApplicationInfo{ee9a2df org.linphone}
12-06 15:54:16.152  1268  2791 E ActivityTrigger: activityStartTrigger: not whiteListedorg.linphone/org.linphone.CallActivity/3311
12-06 15:54:16.153  1268  2791 D ActivityTrigger: activityResumeTrigger: The activity in ApplicationInfo{ee9a2df org.linphone} is now in focus and seems to be in full-screen mode
12-06 15:54:16.153  1268  2791 E ActivityTrigger: activityResumeTrigger: not whiteListedorg.linphone/org.linphone.CallActivity/3311
12-06 15:54:16.153  1268  2791 D ActivityTrigger: ActivityTrigger activityPauseTrigger 
12-06 15:54:16.163 19113 19113 I Linphone: bellesip_wake_lock_acquire(): Android wake lock acquired [ref=0x3372]
12-06 15:54:16.163 19113 19113 I Linphone: channel [0x77222cfec0]: starting recv background task with id=[3372].
12-06 15:54:16.163 19113 19113 I Linphone: channel [0x77222cfec0]: received [615] new bytes from [UDP://::ffff:192.168.96.106:39603]:
12-06 15:54:16.163 19113 19113 I Linphone: ACK sip:242680@172.17.107.231:49874;transport=udp SIP/2.0
12-06 15:54:16.163 19113 19113 I Linphone: Via: SIP/2.0/UDP 192.168.96.98:5060;rport;branch=z9hG4bKPj53bd14dc-0ad2-4007-ac48-4deedb1fa241
12-06 15:54:16.163 19113 19113 I Linphone: From: "28149" <sip:28149@192.168.96.106>;tag=020c9906-f535-46f5-b686-b2699a64ceea
12-06 15:54:16.163 19113 19113 I Linphone: To: <sip:242680@172.17.107.231;app-id=475732041065;pn-type=firebase;pn-tok=dU4XX-Q-2qk:APA91bFFZRO3Y9yswxHzFuUP5OjHuOFhXTtGo0zupvdqDmJ6I2JbGDcnDGBAnIJLdNsGP6Ol46Fg11krjjTJQJ66aWy8WuQPvjJ5WpSAba72G1HPNj3TQk08-_hAnZBYBigDyMlOSfUJ;pn-silent=1>;tag=gnSyxdB
12-06 15:54:16.163 19113 19113 I Linphone: Call-ID: f8f74801-0a13-4a9e-86d1-f5edcb085655
12-06 15:54:16.163 19113 19113 I Linphone: CSeq: 31713 ACK
12-06 15:54:16.163 19113 19113 I Linphone: Max-Forwards: 70
12-06 15:54:16.163 19113 19113 I Linphone: User-Agent: UZPBX
12-06 15:54:16.163 19113 19113 I Linphone: Content-Length:  0
12-06 15:54:16.163 19113 19113 I Linphone: 
12-06 15:54:16.163 19113 19113 I Linphone: 
12-06 15:54:16.166   534 21065 D soundtrigger: audio_extn_sound_trigger_update_stream_status: uc_id 1 of type 0 for Event 2, with Raise=0
12-06 15:54:16.166   534 21065 E msm8974_platform: platform_check_backends_match: Invalid snd_device = 
12-06 15:54:16.166   534 21065 D audio_hw_primary: check_usecases_codec_backend:becf: (47) check_usecases curr device: voice-handset, usecase device: backends match 0
12-06 15:54:16.166   534 21065 D audio_hw_primary: check_usecases_codec_backend:becf: check_usecases num.of Usecases to switch 1
12-06 15:54:16.166   534 21065 D hardware_info: hw_info_append_hw_type : device_name = handset
12-06 15:54:16.166   534 21065 D audio_hw_primary: disable_snd_device: snd_device(1: handset)
12-06 15:54:16.166   534 21065 D msm8974_platform: platform_split_snd_device: snd_device(1) num devices(0) new_snd_devices(0)
12-06 15:54:16.168 15246 15268 D NetworkStateUtils: netWorkState=1
12-06 15:54:16.168 19113 19113 I Linphone: channel [0x77222cfec0] [615] bytes parsed
12-06 15:54:16.168 19113 19113 I Linphone: Incoming INVITE has ACK, dialog is happy
12-06 15:54:16.168 19113 19113 I Linphone: channel [0x77222cfec0]: ending recv background task with id=[3372].
12-06 15:54:16.169 19113 19113 I Linphone: bellesip_wake_lock_release(): Android wake lock released [ref=0x3372]
12-06 15:54:16.169   534 21065 I soundtrigger: audio_extn_sound_trigger_update_device_status: device 0x1 of type 0 for Event 0, with Raise=0
12-06 15:54:16.169   534 21065 D hardware_info: hw_info_append_hw_type : device_name = voice-handset
12-06 15:54:16.169   534 21065 D msm8974_platform: platform_split_snd_device: snd_device(15) num devices(0) new_snd_devices(0)
12-06 15:54:16.169   534 21065 D audio_hw_primary: enable_snd_device: snd_device(15: voice-handset)
12-06 15:54:16.169   534 21065 I soundtrigger: audio_extn_sound_trigger_update_device_status: device 0xf of type 0 for Event 1, with Raise=0
12-06 15:54:16.169 19113 19113 W MediaPlayer: mediaplayer went away with unhandled events
12-06 15:54:16.173   534 21065 D audio_hw_primary: check_usecases_codec_backend:becf: enabling usecase (low-latency-playback) on (voice-handset)
12-06 15:54:16.173   534 21065 D soundtrigger: audio_extn_sound_trigger_update_stream_status: uc_id 1 of type 0 for Event 3, with Raise=0
12-06 15:54:16.173   534 21065 D audio_hw_utils: audio_extn_utils_send_app_type_cfg: usecase->out_snd_device voice-handset
12-06 15:54:16.173   534 21065 D msm8974_platform: platform_split_snd_device: snd_device(15) num devices(0) new_snd_devices(0)
12-06 15:54:16.173   534 21065 E msm8974_platform: platform_get_snd_device_backend_index: BE DAI Name Table is not present
12-06 15:54:16.173   534 21065 E audio_hw_utils: send_app_type_cfg_for_device: Couldn't get the backend index for snd device voice-handset ret=-14
12-06 15:54:16.173   534 21065 I audio_hw_utils: send_app_type_cfg_for_device PLAYBACK app_type 69937, acdb_dev_id 7, sample_rate 48000, snd_device_be_idx -14
12-06 15:54:16.173   534 21065 D msm8974_platform: platform_split_snd_device: snd_device(15) num devices(1) new_snd_devices(0)
12-06 15:54:16.173   534 21065 D ACDB-LOADER: ACDB -> send_audio_cal, acdb_id = 7, path = 0, app id = 0x11131, sample rate = 48000
12-06 15:54:16.173   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AUDPROC_GAIN_DEP_STEP_TABLE, vol index 0
12-06 15:54:16.173   534 21065 D ACDB-LOADER: ACDB -> AUDIO_SET_VOL_CAL cal type = 12
12-06 15:54:16.173   534 21065 D ACDB-LOADER: ACDB -> audstrm_cal->cal_type.cal_data.cal_size = 16
12-06 15:54:16.173   534 21065 D ACDB-LOADER: ACDB -> GET_AFE_TOPOLOGY_ID for adcd_id 7, Topology Id 112fc
12-06 15:54:16.173   534 21065 D         : Failed to fetch the lookup information of the device 00000007 
12-06 15:54:16.173   534 21065 E ACDB-LOADER: Error: ACDB_CMD_GET_AFE_COMMON_TABLE_SIZE Returned = -19
12-06 15:54:16.174   534 21065 D         : Failed to fetch the lookup information of the device 00000007 
12-06 15:54:16.174   534 21065 E ACDB-LOADER: Error: ACDB AFE returned = -19
12-06 15:54:16.174   534 21065 D ACDB-LOADER: ACDB -> send_hw_delay : acdb_id = 7 path = 0
12-06 15:54:16.174   534 21065 D audio_hw_primary: enable_audio_route: apply mixer and update path: low-latency-playback
12-06 15:54:16.174 19113 19113 D AppTracker: App Event: stop
12-06 15:54:16.178 19113 19113 I Linphone: Vtable [0x771f313200] unregistered on core [0x772dc55a00]
12-06 15:54:16.178 19113 19113 I Linphone: Vtable [0x771f312e40] unregistered on core [0x772dc55a00]
12-06 15:54:16.179 19113 19113 I Linphone: Activity paused:org.linphone.CallIncomingActivity@5f2b7af
12-06 15:54:16.179 19113 19113 I Linphone: runningActivities=0
12-06 15:54:16.180  1268 12140 D ActivityTrigger: activityResumeTrigger: The activity in ApplicationInfo{ee9a2df org.linphone} is now in focus and seems to be in full-screen mode
12-06 15:54:16.180  1268 12140 E ActivityTrigger: activityResumeTrigger: not whiteListedorg.linphone/org.linphone.CallActivity/3311
12-06 15:54:16.181  1268 12140 V KeyguardServiceDelegate: setOccluded(false) animate=false
12-06 15:54:16.181  1268 12140 I ActivityManager: [scene] setLockScreenShown showing false, occluded false
12-06 15:54:16.181  1268  1268 D OemSceneButtonController: [scene] start monitoring, content://settings/system/game_mode_status content://settings/system/game_mode_lock_buttons
12-06 15:54:16.184  1970  2170 D KeyguardViewMediator: setOccluded false
12-06 15:54:16.185  1970  1970 D KeyguardViewMediator: handleSetOccluded false
12-06 15:54:16.185  1970  1970 D StatusBarWindowManager: applyFocusableFlag: true, false, false, 81840068
12-06 15:54:16.185  1970  1970 D KeyguardViewMediator: adjustStatusBarLocked: mShowing=false mOccluded=false isSecure=true --> flags=0x0
12-06 15:54:16.188   908 21042 D NuPlayerDriver: notifyListener_l(0xe9132460), (2, 0, 0, -1), loop setting(0, 0)
12-06 15:54:16.188  1268  1284 I MediaFocusControl:  AudioFocus  abandonAudioFocus() from uid/pid 10027/1970 clientId=android.media.AudioManager@de9755c
12-06 15:54:16.192 19113 19113 I Linphone: Activity created:org.linphone.CallActivity@d365402
12-06 15:54:16.192  1268  1284 D ActivityManager: setRequestedOrientation() to 1 for ActivityRecord{f9d76c5 u0 org.linphone/.CallActivity t2277} from pid=19113, uid=10339
12-06 15:54:16.214   534 21065 D hardware_info: hw_info_append_hw_type : device_name = voice-handset
12-06 15:54:16.214   534 21065 I msm8974_platform: platform_check_and_set_capture_codec_backend_cfg:txbecf: afe: bitwidth 16, samplerate 48000, channel 1 format 1, backend_idx 7 usecase = 32 device (voice-dmic-ef)
12-06 15:54:16.214   534 21065 I msm8974_platform: platform_check_capture_codec_backend_cfg:txbecf: afe: Codec selected backend: 7 current bit width: 16 and sample rate: 48000, channels 1 format 1
12-06 15:54:16.214   534 21065 W msm8974_platform: platform_check_capture_codec_backend_cfg:txbecf: afe: Use default bw and sr for voice/voip calls and for unprocessed/camera source
12-06 15:54:16.214   534 21065 I msm8974_platform: platform_check_capture_codec_backend_cfg:txbecf: afe: Codec selected backend: 7 updated bit width: 16 and sample rate: 48000
12-06 15:54:16.214   534 21065 D audio_hw_primary: check_usecases_capture_codec_backend:becf: force routing 0
12-06 15:54:16.214   534 21065 D hardware_info: hw_info_append_hw_type : device_name = voice-dmic-ef
12-06 15:54:16.214   534 21065 D msm8974_platform: platform_split_snd_device: snd_device(92) num devices(0) new_snd_devices(0)
12-06 15:54:16.214   534 21065 D audio_hw_primary: enable_snd_device: snd_device(92: voice-dmic-ef)
12-06 15:54:16.214   534 21065 I soundtrigger: audio_extn_sound_trigger_update_device_status: device 0x5c of type 1 for Event 1, with Raise=1
12-06 15:54:16.214   534 21065 D sound_trigger_platform: platform_stdev_check_and_update_concurrency: concurrency active 0, tx 1, rx 0, concurrency session_allowed 0
12-06 15:54:16.227   534 21065 D ACDB-LOADER: ACDB -> send_voice_cal, acdb_rx = 7, acdb_tx = 41, feature_set = 0
12-06 15:54:16.227   534 21065 D ACDB-LOADER: ACDB -> send_voice_rx_topology
12-06 15:54:16.227   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_VOCPROC_COMMON_TOPOLOGY_ID
12-06 15:54:16.227   534 21065 D ACDB-LOADER: ACDB -> send_voice_tx_topology
12-06 15:54:16.227   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_VOCPROC_COMMON_TOPOLOGY_ID
12-06 15:54:16.227   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_AFE_DATA
12-06 15:54:16.227   534 21065 D ACDB-LOADER: ACDB -> AUDIO_SET_SIDETONE_CAL
12-06 15:54:16.227   534 21065 D ACDB-LOADER: ACDB -> send_voice_columns, rxacdb_id 7, txacdb_id 41, table 5
12-06 15:54:16.227   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_VOC_COLUMNS_INFO_V2
12-06 15:54:16.227   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_VOC_PROC_STATIC_TABLE_SIZE
12-06 15:54:16.228   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_VOC_PROC_STATIC_TABLE
12-06 15:54:16.228   534 21065 D ACDB-LOADER: ACDB -> AUDIO_SET_VOCPROC_CAL
12-06 15:54:16.228   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_VOC_PROC_DEVICE_CFG
12-06 15:54:16.228   534 21065 D ACDB-LOADER: ACDB -> AUDIO_SET_VOCPROC_DEV_CFG_CAL
12-06 15:54:16.228   534 21065 D ACDB-LOADER: ACDB -> send_voice_columns, rxacdb_id 7, txacdb_id 41, table 4
12-06 15:54:16.228   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_VOC_COLUMNS_INFO_V2
12-06 15:54:16.228   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_VOC_PROC_DYNAMIC_TABLE_SIZE
12-06 15:54:16.230   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_VOC_PROC_DYNAMIC_TABLE
12-06 15:54:16.233   534 21065 D ACDB-LOADER: ACDB -> AUDIO_SET_VOCPROC_VOL_CAL
12-06 15:54:16.233   534 21065 D ACDB-LOADER: ACDB -> send_voice_columns, rxacdb_id 7, txacdb_id 41, table 6
12-06 15:54:16.233   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_VOC_COLUMNS_INFO_V2
12-06 15:54:16.233   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_VOC_STREAM_STATIC_TABLE_SIZE
12-06 15:54:16.233   534 21065 D ACDB-LOADER: ACDB -> ACDB_CMD_GET_VOC_STREAM_STATIC_TABLE
12-06 15:54:16.233   534 21065 D ACDB-LOADER: ACDB -> AUDIO_SET_VOCPROC_STREAM_CAL
12-06 15:54:16.234   534 21065 D ACDB-LOADER: ACDB -> GET_AFE_TOPOLOGY_ID for adcd_id 41, Topology Id 112fb
12-06 15:54:16.234   534 21065 D         : Failed to fetch the lookup information of the device 00000029 
12-06 15:54:16.234   534 21065 E ACDB-LOADER: Error: ACDB_CMD_GET_AFE_COMMON_TABLE_SIZE Returned = -19
12-06 15:54:16.234   534 21065 D         : Failed to fetch the lookup information of the device 00000029 
12-06 15:54:16.234   534 21065 E ACDB-LOADER: Error: ACDB AFE returned = -19
12-06 15:54:16.234   534 21065 D ACDB-LOADER: ACDB -> GET_AFE_TOPOLOGY_ID for adcd_id 7, Topology Id 112fc
12-06 15:54:16.234   534 21065 D         : Failed to fetch the lookup information of the device 00000007 
12-06 15:54:16.234   534 21065 E ACDB-LOADER: Error: ACDB_CMD_GET_AFE_COMMON_TABLE_SIZE Returned = -19
12-06 15:54:16.234   534 21065 D         : Failed to fetch the lookup information of the device 00000007 
12-06 15:54:16.234   534 21065 E ACDB-LOADER: Error: ACDB AFE returned = -19
12-06 15:54:16.234   534 21065 D ACDB-LOADER: ACDB -> send_hw_delay : acdb_id = 41 path = 1
12-06 15:54:16.234   534 21065 D ACDB-LOADER: ACDB -> send_hw_delay : acdb_id = 7 path = 0
12-06 15:54:16.234   534 21065 D ACDB-LOADER: ACDB -> Sent VocProc Cal!
12-06 15:54:16.234   534 21065 E audio_hw_utils: audio_extn_utils_update_stream_app_type_cfg_for_usecase: app type cfg not supported for usecase type (3)
12-06 15:54:16.234   534 21065 D soundtrigger: audio_extn_sound_trigger_update_stream_status: uc_id 32 of type 3 for Event 3, with Raise=0
12-06 15:54:16.234   534 21065 I audio_hw_utils: audio_extn_utils_send_app_type_cfg: not a playback/capture path, no need to cfg app type
12-06 15:54:16.234   534 21065 D audio_hw_primary: enable_audio_route: apply mixer and update path: compress-voip-call
12-06 15:54:16.235   534 21065 D audio_hw_primary: select_devices: done
12-06 15:54:16.235   534 21065 D compress_voip: voip_start_call: Opening PCM capture device card_id(0) device_id(3)
12-06 15:54:16.236   534 21065 D compress_voip: voip_start_call: Opening PCM playback device card_id(0) device_id(3)
12-06 15:54:16.289  1970  2215 D SystemServicesProxy: getRecentTasks: org.linphone, 2277, false, 0
12-06 15:54:16.290  1970  2215 D SystemServicesProxy: getRecentTasks: pl.solidexplorer2, 2278, false, 0
12-06 15:54:16.310   534 21065 D voice   : voice_set_sidetone: enable, out_snd_device: 15
12-06 15:54:16.319  3081 21069 E libc    : Access denied finding property "net.dns1"
12-06 15:54:16.320   471   471 W auditd  : type=1400 "Thread-221""u:object_r:net_dns_prop:s0""tmpfs"
12-06 15:54:16.362   471   471 W auditd  : type=1400 "Thread-221""interrupts""proc"
12-06 15:54:16.384 19113 19113 I Linphone: Activity started:org.linphone.CallActivity@d365402
12-06 15:54:16.388 19113 19113 I Linphone: Activity resumed:org.linphone.CallActivity@d365402
12-06 15:54:16.388 19113 19113 I Linphone: runningActivities=1
12-06 15:54:16.388 19113 19113 I Linphone: Core callbacks [0x77229e3890] registered on core [0x772dc55a00]
12-06 15:54:16.390   893  1245 I APM_AudioPolicyManager: stopOutput() output 13, stream 5, session 265
12-06 15:54:16.391   893  1245 V AudioPolicyManagerCustom: handleIncallSonification() stream 5 starting 0 device 1 stateChange 0
12-06 15:54:16.391   893  1245 V AudioPolicyManagerCustom: handleIncallSonification() low visibility, muteCount 1
12-06 15:54:16.391   893  1245 I AudioPolicyManagerCustom: checkAndSetVolume:stream:5,index:5 device:1,delayMs:0,muteCount:0
12-06 15:54:16.391   893  1244 I AudioFlinger: setStreamVolume,stream:5,value:0.188366
12-06 15:54:16.391   893  1245 I APM_AudioPolicyManager: setOutputDevice() device 0001 delayMs 96

**Comment by crontab888** [comment]
Yep, i am getting exactly the same -- tried on android 6, 7, 7.1 and 8.
As a note, incoming call comes in, no audio, i place the call on hold form any A or B party, after off hold, audio starts working -- maybe it is the same for you?
Banged my head around a bit, could not find a way forward.

**Comment by totakura** [comment]
I found this problem with my setup too; it used to work before. As explained by @crontab888, the incoming audio plays after placing the call in hold and resuming.
I'm using Asterisk 13 as a SIP server in my setup.

Using `adb logcat "Linphone:V *:S"`, I found the following lines interesting:

    W Linphone: Excess of audio samples in capture side bytes=640 (20 ms)
    ...
    W Linphone: Flow controlled bufferizer of max 40 ms is filled with 1230 ms at the end of the 5000 ms interval, asking to drop 595 ms
    W Linphone: Too much buffered audio signal, throwing out 595 ms
    ...

These messages continue to appear.

**Comment by miloscode** [comment]
I am getting the same issue here. Linphone Android 3.3.1. Audio on incoming calls works only after placing the call in hold and resuming. Any ideas how to overcome this?

**Comment by pablojan** [comment]
Same issue here, Android 6.01, CM-13.0, Linphone Android 3.3.1

**Comment by anmol1991** [comment]
same issue here also. 

One more issue, During creation of new sip account, progress bar doesn't dismiss and keeps on going.(Linphone Android 3.3.1)

I guess the error is in registrationState() method of AssistantActivity.java file. 
address.asString().equals(cfg.getAddress().asString()) is coming false whenever there is some character in display name and progress.dismiss() is never being called.

**Comment by whitetornado** [comment]
i got the same issue only when my phone sleeps (several motorola e4 and last build 3.3.1) there is no incoming call but a error for the caller. When the phones wakes-up it can recieve calls.
|
background mode on
enable service notificaton on
enable push notifications off
Only wifi off
phones not use wifi but 3/4g data

**Comment by emilianoOlivier** [comment]
same issue on incoming call.
Anyone have found a workaround to this?
the interesting line i found ( i don't know if useful) are these:

V/AudioHardwareALSA( 1471): setParameters() Turn on Voip
E/audio_a2dp_hw( 1471): adev_set_parameters: ERROR: set param called even when stream out is null
D/AudioPolicyManagerBase( 1471): getInput() inputSource 7, samplingRate 48000, format 1, channelMask 10, acoustics 0
E/legacy_audio_hw_hal( 1471): int android_audio_legacy::adev_open_input_stream(audio_hw_device*, audio_io_handle_t, audio_devices_t, audio_config*, audio_stream_in**): devices before=0x80000010
E/legacy_audio_hw_hal( 1471): int android_audio_legacy::adev_open_input_stream(audio_hw_device*, audio_io_handle_t, audio_devices_t, audio_config*, audio_stream_in**): devices after=0x100000

**Comment by emilianoOlivier** [comment]
as a momentary fix (as a first step) i have add in LinphoneManager  where the icoming call is connected this line:
LinphoneManager.getLc().enableSpeaker(true);

**Comment by Koutoulakis** [comment]
Hi! 
The problem might have happened because of  the mediastreamer submodule. If you downgrade it, (and the linphone submodule, for 3 versions, so that they are compatible and compile correctly)  it works properly.

**Comment by AndySchroder** [comment]
I'm having this problem with my motorola XT1095 running CM13. On my HTC One running CM12, there is audio, but the quality is very bad on incoming calls.

I'm using Linphone version 3.3.1 from https://linphone.org/releases/android/linphone-latest.apk

**Comment by xpresslogistics2015** [comment]
Hi was issue resolved?

**Comment by xpresslogistics2015** [comment]
> as a momentary fix (as a first step) i have add in LinphoneManager where the icoming call is connected this line:
> LinphoneManager.getLc().enableSpeaker(true);

Hi, pardon me I am a learner, where excatly would I put this ? Thanks

**Comment by yudhisthira123** [comment]
Hello, 

there is no audio for either incoming or outgoing call on my HTC U12+ which has inbuilt echo canceller. 

sample code for disable software echo canceller is:-

if(linphoneCore.hasBuiltinEchoCanceller()) {
      linphoneCore.enableEchoCancellation(false);
} else {
      linphoneCore.enableEchoCancellation(true);
}

please help me.

regards,
Yudhi
